cfn/README-GD.md

```diff
@@ -66,12 +70,71 @@ Make a note of the `access_key_id` and `secret_key` values, which you need when
 curl -s -X GET -H "x-aims-auth-token: $AL_TOKEN" https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys | jq
 ```
 
-**Use the selected access_key_id in the following curl command to delete the key:**
+**Use the selected access_key_id in the following command to delete the key:**
 
 ```
 curl -X DELETE -H "x-aims-auth-token: $AL_TOKEN" https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys/<ACCESS_KEY_ID_HERE>
 ```
 
+
+### Windows
+
+The instruction depends on which version of PowerShell you have installed.
+You can check it by running PowerShell and typing in:
```

Comment:
I'd drop line 82 and have line 83 say something like this:
"Check the version of PowerShell on your system:"